config/multi_agent_zhong.json

```diff
@@ -1,109 +1,118 @@
 {
-  "scenario": 
-  {
-      "shape": [50, 50],
-      "wrapped": true,
-      "default_delay": "inertial",
-      "default_cell_type": "hoya",
-  
-      "default_state": 
-      {
-          "cell_id": "DefaultCell",
-          "population": 100,
-          "susceptible": 1,
-          "infected": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-          "recovered": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-          "num_infected": 16,
-          "num_recovered": 30,
-          "initial_infected": 0,
-          "border_length": 1,
-          "land_area": 0,
-          "neighborhood": []
-      },
-  
-      "default_config": 
-      {
-          "hoya": 
-          {
-            "virulence": 0.6,
-            "recovery": 0.4
-          }
-      }
-  },
+    "scenario": {
+        "default_delay": "inertial",
+        "default_cell_type": "zhong",
+        "default_state": {
+            "population_density": 1,
+            "susceptible": 1,
+            "infected": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+            "recovered": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        },
+        "default_vicinity": {
+            "correlation": 0
+        },
+        "default_config": {
+            "zhong": {
 
-  "cells":
-  [
-      {
-          "cell_id": "Country1",
-          "susceptible": 0.7,
-          "infected": [0.3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-          "recovered": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-          "num_infected": 16,
-          "num_recovered": 30,
-          "initial_infected": 0,
-          "border_length": 100,
-          "land_area": 100,
-          "neighborhood":
-          [
-              {
-                  "cell_id": "Country2",
-                  "shared_border_length": 500,
-                  "number_shared_roads": 5
-              },
-              {
-                  "cell_id": "Country3",
-                  "shared_border_length": 10,
-                  "number_shared_roads": 2
-              }
-          ]
-      },
-      {
-          "cell_id": "Country2",
-          "susceptible": 1,
-          "infected": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-          "recovered": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-          "num_infected": 16,
-          "num_recovered": 30,
-          "initial_infected": 0,
-          "border_length": 100,
-          "land_area": 700,
-          "neighborhood":
-          [
-              {
-                "cell_id": "Country1",
-                "shared_border_length": 500,
-                "number_shared_roads": 5
-              },
-              {
-                "cell_id": "Country3",
-                "shared_border_length": 20,
-                "number_shared_roads": 10
-              }
-          ]
-      },
-      {
+                "virulency_rates": [[0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15],
```

Comment:
Shouldn't this be divided into two vectors, `virulence_rates` and `recovery_rates` ?

Reply:
Each vector represents a phase. So in this case since there are two vectors, there are two phases. Though it should be noted that this is an example- in the code (ie not the JSON file), only the first phase is used.